pkg/prowloader/prow.go

```diff
@@ -420,6 +422,24 @@ func (pl *ProwLoader) extractTestCases(suite *junit.TestSuite, testCases map[str
 			testNameWithKnownSuite = fmt.Sprintf("%s.%s", suite.Name, tc.Name)
 		}
 
+		if failureOutput != nil {
+			// Check if this test is configured to extract metadata from it's output, and if so, create it
+			// in the db.
+			extractedMetadata := testOutputMetadataExtractor.ExtractMetadata(testNameWithKnownSuite, failureOutput.Output)
+			if len(extractedMetadata) > 0 {
+				failureOutput.Metadata = make([]models.ProwJobRunTestOutputMetadata, 0, len(extractedMetadata))
+				for _, m := range extractedMetadata {
+					jsonb := pgtype.JSONB{}
+					if err := jsonb.Set(m); err != nil {
+						panic(err)
```

Comment:
Can we log a warning  instead? I don't know if we'll really ever panic on `jsonb.Set` but the places where we panic during syncing risks data loss since it'll stop syncing and any further tasks.